imgops/README.md

```diff
@@ -1,4 +1,4 @@
-# Img Opts for the Grid
+# ImgOps for the Grid
 Local version of the imgops service
 
 ## Requirements
```

Comment:
We should add Docker to the requirements

Reply:
https://github.com/guardian/grid/blob/master/dev-start.sh#L27 😄

Reply:
updated the setup docs now too.